Makefile

```diff
@@ -1,4 +1,4 @@
-PACKAGES ?= mountinfo mount sequential signal symlink user
+PACKAGES ?= mountinfo mount sequential signal symlink user userns # IMPORTANT: when updating this list, also update the conditional one in .github/workflows/test.yml
```

Comment:
And added a comment here as a reminder to check the one in the GitHub actions workflow when updating.